src/garage/torch/utils.py

```diff
@@ -44,3 +44,20 @@ def flatten_batch(tensor):
 
     """
     return tensor.reshape((-1, ) + tensor.shape[2:])
+
+
+def update_module_params(module, new_params):
```

Comment:
@krzentner can you comment on the torchi-ness of this approach? who uses it?

if this is never intended to be used by a package user, please make sure it's an _private function.

Reply:
People almost never do this. Except for the weird case were you need your parameters to be in the middle of the tape, you can just use `module.load_state_dict()`.

Reply:
Having said that, this function does basically do the same thing as `load_state_dict()` (with the minor different that it doesn't erase all of the parameter `grad` / `grad_fn`), so perhaps we should consider it fairly torchy?